include/cpr/range.h

```diff
@@ -11,8 +11,30 @@ class Range {
 
     std::int64_t resume_from = 0;
     std::int64_t finish_at = 0;
+
+    const std::string str() const {
+        std::string from_str = (resume_from == -1) ? "" : std::to_string(resume_from);
```

Comment:
Where is `resume_from` set to `-1`? Since I was thinking about replacing `resume_from == -1` with `resume_from < 0`.
This is a safer alternative to check for ranges while we do not ECC-RAM everywhere :)

Reply:
-1 is the value in the documentation for creating partial ranges. Nevertheless, < 0 is the safer alternative.